prql-compiler/src/error.rs

```diff
@@ -137,7 +137,7 @@ impl Display for ErrorMessage {
 
             writeln!(f, "{}Error: {}", code, &self.reason)?;
             if let Some(hint) = &self.hint {
-                writeln!(f, " Hint: {}", hint)?;
+                writeln!(f, "Hint: {}", hint)?;
```

Comment:
@aljazerzen lmk if there was a reason for this and I'll revert

Reply:
I've added that for visual grouping of non-composed errors:

```
Error: blah
Hint: bleh
Error: blah
Hint: bleh
```
... looks like these are 4 separate messages.

Reply:
Added #2675 as a synthesis...